ledger/ledgercore/error.go

```diff
@@ -55,7 +55,7 @@ func MakeLeaseInLedgerError(txid transactions.Txid, lease Txlease) *LeaseInLedge
 func (lile *LeaseInLedgerError) Error() string {
 	// format the lease as address.
 	addr := basics.Address(lile.lease.Lease)
-	return fmt.Sprintf("transaction %v using an overlapping lease %s", lile.txid, addr.String())
+	return fmt.Sprintf("transaction %v using an overlapping lease (%s, %s)", lile.txid, lile.lease.Sender.String(), addr.String())
```

Comment:
this is very confusing :) maybe rename addr to `leaseData`

Reply:
What do you mean? the "lease" in actual string to "(sender, lease)"

Reply:
yes, but `addr := basics.Address(lile.lease.Lease)` this is not address but the lease value. I'm referencing to `addr.String()` that prints lease data, not addr as one can think

Reply:
done, I think